src/content/docs/products/splashkit/SplashKit Tutorials/0-overview.mdx

```diff
@@ -1,9 +1,41 @@
 ---
-title: Overview
+title: Tutorials Overview
+description: The SplashKit tutorials team works on the creation of tutorials using the SplashKit library.
 sidebar:
   order: 0
 ---
 
 import { Card, LinkCard, CardGrid, Icon } from "@astrojs/starlight/components";
 
-Placeholder page.
+The SplashKit tutorials team works on the creation of tutorials using the SplashKit library. The team is responsible for creating tutorials that cover a wide range of topics, from Graphics, Interfaces, Networking and Raspberry Pis. The tutorials are designed to help users learn how to use SplashKit and to provide them with the knowledge they need to create their own projects. Each tutorial contains an explanation of the topic, the steps involved, as well as code snippets done in C#, C++, Python and Pasal.
+
+## Tutorial Topics
+
+The current list of tutorial topics includes:
+
+- Animations
+- Audio
+- Camera
+- Graphics
+- Input
+- Interface
+- JSON
+- Networking
+- Raspberry GPIO
+- Resource Bundles
+- Sprites
+- Utilities
+
+## The current Documentation
+
+1. [Tutorial Proposal Template](/products/splashkit/splashkit-tutorials/documentation/1-tutorial-proposal-template) The template for creating a tutorial proposal.
+2. [Tutorial Style Guide](/products/splashkit/splashkit-tutorials/documentation/2-tutorial-style-guide) The style guide for creating tutorials.
+3. [Tutorial Reviews](/products/splashkit/splashkit-tutorials/documentation/3-tutorial-reviews) The current tutorials that need to be reviewed.
+
+## The current Onboarding Documentation
```

Comment:
Recommend keeping this to SplashKit Tutorials onboarding. The generic SplashKit onboarding stuff should sit in the SplashKit parent page. 

_Scenario_
Day 1 - New student joins SplashKit. New student unsure which project to pick yet, but knows they want to work with SplashKit. Works on the SplashKit onboarding to get what is common between all projects.
Day 3 - New student decides to join the SplashKit Tutorials team. They now need to extend their onboarding to be specific to the project.
Day 5 - New student is industrious and wants to also join the SplashKit Website team. Their onboarding will have the information specific to them. But they have all of the core information they need from the SplashKit general onboarding.

Reply:
Currently there is no onboarding section on the website, so this could potentially be moved later into a dedicated onboarding hub. But for now the people in the tutorials team are needing these to finish set up and to also know how to do commit pull requests and peer reviews.

This could be moved into an onboarding hub before trimester 3 instead.